src/main/java/core/basesyntax/TreeSetCharacters.java

```diff
@@ -17,7 +23,30 @@
  * Результат 2: acf</p>
  */
 public class TreeSetCharacters {
-    public String getUniqueCharacters(String fileName) {
-        return null;
+    public String getUniqueCharacters(String fileName) throws FileNotFoundException {
+        StringBuilder stringBuilder = new StringBuilder();
+        TreeSet treeSet = new TreeSet();
+        String result = "";
+
+        try (FileReader fileReader = new FileReader(fileName)) {
+            BufferedReader bufferedReader = new BufferedReader(fileReader);
+            int symbol = bufferedReader.read();
+            while (symbol != -1) {
+                treeSet.add((char) Character.toLowerCase(symbol));
+                symbol = bufferedReader.read();
+            }
+        } catch (IOException e) {
+            throw new FileNotFoundException("There is no file!");
+        }
+
+        for (Object o : treeSet) {
+            stringBuilder.append(o);
```

Comment:
let's stop appending when(if) stringBuilder.length() == 5 and not take substring after